### DISCUSSION TITLE
2023-01-25-my-2022

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#132] [General]
# 2022 年总结 - 率性而活 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share今天是 2023 年的大年初四，对于拖延症患者而言再不写可能就被我拖没了，那可不行，等到老了以后再来回忆应该都记不住发生了啥，对我而言 2022 年是值得记录的一年，本文打算从生活、折腾、工作三方面来随便聊聊。

http://localhost:4000/2023-01-25/2022-summary.html

**Comment by andy660** [comment]
很喜欢up主的内容。作为纯小白专门开了个github账号来学习。谢谢！

**Reply by tw93** [reply]
谢谢喜欢

**Comment by austin2035** [comment]
感觉楼主过得充实而有收获，慕了，向你学习。

**Reply by tw93** [reply]
👋  加油加油  需要过得充实些，重复的日子我看来就是浪费生命了

**Comment by MaoVlog** [comment]
非常喜欢中这博客。这评论系统用的是giscus?

**Reply by tw93** [reply]
哈哈 是的 不过被我改了改 整体和博客统一点儿

**Reply by AAChartModel** [reply]
有考虑过为博客增加夜间模式吗? 喜欢晚上看😊

**Comment by Liqishun** [comment]
每次阅读都有收获，持续关注

**Comment by HeroadZ** [comment]
真好，能感受到博主认真生活的态度，受益良多，感谢。

**Comment by hczs** [comment]
很喜欢博主的中英文字体和文章排版，看着真舒服！

**Reply by ingorewho** [reply]
每一天应该不一样一些，赞同

**Comment by liucoffe2** [comment]
吃辣椒的话，去衢州会比较爽。

**Reply by tw93** [reply]
有去过那边，衢州烧烤挺不错

**Comment by liucoffe2** [comment]
作为湖南人，我更喜欢辣椒。

Tw93 ***@***.***>于2023年3月8日 周三21:36写道：

**Comment by zimiyu** [comment]
大佬也喜欢李先生啊，太好了~

**Reply by tw93** [reply]
哈哈 特别喜欢李志

**Comment by yinzhidong** [comment]
hello

**Reply by tw93** [reply]
World

**Comment by yujiachen-y** [comment]
感觉大佬的效率好高，带团队 + 做开源 + 写周刊，好奇有啥经验可以分享的吗？

**Reply by tw93** [reply]
把刷抖音玩游戏的业余时间全部抽出来做自己喜欢的事情，然后✖️ 365 就可以做很多事事情了

**Comment by liucoffe2** [comment]
tw93是我见过的超有灵气、超级勤奋聪敏、又不乏生活情趣的科学家。

Tw93 ***@***.***>于2024年2月13日 周二15:17写道：

**Reply by tw93** [reply]
客气了，不是科学家，就是一个敲代码的程序员，哈哈
